web/terraform/main.tf

```diff
@@ -53,23 +53,20 @@ resource "azurerm_resource_group" "resource-group" {
   tags     = local.common-tags
 }
 
-resource "azurerm_app_service_plan" "education-benchmarking-asp" {
+resource "azurerm_service_plan" "education-benchmarking-asp" {
```

Comment:
Will destroying the old service plan affect the dependent app service?

Reply:
Yes, there will need to be some manual intervention as there is a "clean" migration path,